lib/drivers/tandemTslimDriver.js

```diff
@@ -1345,10 +1346,10 @@ module.exports = function (config) {
             console.log('COMMAND BASAL NOT EQUAL TO BASE!!');
           }
           postbasal = cfg.builder.makeScheduledBasal()
-            .with_scheduleName(_.find(data.profiles,{idp:event.idp}).name)
             .with_deviceTime(event.deviceTime)
             .with_rate(event.command_basal_rate)
-            .set('index', event.index);
+            .set('index', event.index)
+            .set('idp', event.idp);
```

Comment:
We don't want to upload the profile index `idp` on all basals. We should either put it in the payload or clean it up in the simulator prior to uploading.